### PR TITLE
Install `mattermost-redux` from latest master

### PR DESCRIPTION
**Pull request by lauraseidler**
Hi @SrMouraSilva,

not sure if you're accepting PRs on this, but we've been using your plugin for a while now, and noticed it stopped working after a recent Mattermost update (posts never being sent and always staying in pending state). Our Mattermost users were very sad that it wasn't working anymore, so I've tracked down the problem, you can find more info on it here: mattermost/mattermost-redux#1425

TL;DR: The latest Mattermost web app doesn't include `redux-offline` anymore, so `createPost`, which this plugin also uses, doesn't work anymore. Unfortunately there's no released version without `redux-offline`, so a working version has to be installed from GitHub directly.

We've been running this for a few days now without problems, so I thought you might wanna merge it in.

Cheers!

### CODE REVIEW

**Comment by SrMouraSilva** [comment]
Hello @lauraseidler. Thanks for the MR.

Here, we use the Mattermost installed with Gitlab omnibus. Generally, Gitlab uses a previous Mattermost version: https://docs.gitlab.com/omnibus/gitlab-mattermost/#upgrading-gitlab-mattermost

We currently are using the Mattermost 5.33. Today we will update the Gitlab, then, we will use 5.34.

I don't now if this problem occurs only in 5.35 and upper... After update, I will check, test, then accept this MR. :D

Cheers!

**Comment by hmhealey** [comment]
Hi, one of the Mattermost devs here.

The new version of mattermost-redux should actually work with older versions of the web app for what this plugin is using. The only things I can see being used from it are `createPost` (which is broken in the current version of mattermost-redux when using 5.35 and newer) and some theming stuff that hasn't changed in a long time.

The change to remove redux-offline from createPost replaces it with code that's functionally identical, so this PR should work fine with older versions of the web app.

**Comment by SrMouraSilva** [comment]
Hello, @lauraseidler. 

I updated the mattermost and this error also occurs with me.

I migrate this plugin to a new repo https://github.com/SrMouraSilva/mattermost-google-meet-plugin. The new repo mas been generated by the mattermost plugin theme.

In the new repository, I manually applied your change. I reference you into the Readme https://github.com/SrMouraSilva/mattermost-google-meet-plugin#acknowledgment. Thanks for the helping!
